onsite_auth_server/src/interface.ts

```diff
@@ -0,0 +1,58 @@
+// interface.ts Copyright 2020 Manchester Makerspace MIT License
+import { Db, MongoClient } from 'mongodb';
+
+interface cardDataI {
```

Comment:
NIT: Generally types and interfaces are TitleCase. Also, the "I" or "Type" or any reference that it's a type/interface is excluded as its assumed from the TitleCase. This also lends itself well to constructing complex classes from interfaces. `class Foobar implements FizzBuzz`

Reply:
Good to know, that's pretty easy to change